src/App.css

```diff
@@ -1,39 +1,340 @@
+:root {
+  --pale-grey: #f5f6fa;
+  --azure: #0290ff;
+  --black-two: #121212;
+}
+
 .App {
-  text-align: center;
+  display: flex;
+  width: 100%;
+  height: auto;
+  background-color: var(--pale-grey);
+}
+
+.main-wrapper {
+  height: 100%;
+  width: 80%;
+  margin: 0 auto;
+  background-color: #fafafa;
+}
+
+.header {
+  width: 100%;
+  height: 5.938rem;
+  margin: 2rem auto;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.link {
+  margin-right: 3rem;
+}
+
+.logo-navabr {
+  display: flex;
+  justify-content: center;
+}
+
+.logo {
+  width: auto;
+  height: 2.313rem;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
```

Comment:
- [ ] This CSS property is also used more than once, it would be nice to use a reusable class which can be reused on the affected components, kindly do so to stick to [best practices](https://speckyboy.com/good-bad-css-practices/)